scripts/starknet_ibc_core.ts

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env -S deno run
```

Comment:
Just a side note: My formatter (`prettierd`) made a bunch of changes to these `.ts` files. 
Not sure if you’ve run a formatter on them, but since it’s not part of CI, it should be fine. Though, still, good to keep things formatted.

Reply:
Perhaps could run `deno fmt`